loaders/mock.js

```diff
@@ -51,7 +57,7 @@ async function _uploadToDynamo (tableName, item) {
     }
 
     await dynamoDB.put(putParams).promise()
-    return null
+    return response?.Item?.ECurl || null
```

Comment:
`ECurl` is kind of an awkward variable name. What do you think about just writing it out full? `elastiCacheUrl`?